VMD_scripts/Blob_GUI.tcl

```diff
@@ -32,30 +56,62 @@ variable blobs [toplevel ".blob"]
 }
 proc ::blobulator::GUI {} {
 	::blobulator::Window
-	grid [label $::blobulator::blobs.1_MolID -text MolID ]
-	grid [entry $::blobulator::blobs.tv_MolID -width 10 -textvariable ::blobulator::MolID ] -row 0 -column 1
+	grid [label $::blobulator::blobs.1_MolID -text MolID: ] -row 0 -column $::blobulator::textColumn -sticky e
+	grid [entry $::blobulator::blobs.tv_MolID -width 10 -textvariable ::blobulator::MolID ] -row 0 -column $::blobulator::paraColumn -columnspan 1 
 	if {$::blobulator::MolID == ""} {
 		set ::blobulator::MolID "top"
 	}
-	set paraList [list Lmin ::blobulator::Lmin 1 50 1 H ::blobulator::H .1 1 .01]
-	foreach { entry variableName min max interval} $paraList {
-		set w1 [label $::blobulator::blobs.l_$entry -text $entry]
-		set w2 [entry $::blobulator::blobs.e_$entry -width 10 -textvariable $variableName]
-		set w3 [scale $::blobulator::blobs.s_$entry -orient horizontal -from $min -to $max -length 175 -resolution $interval -tickinterval 0 -variable $variableName -showvalue 0]
+
+	#Atomselect grids
+	grid [label $::blobulator::blobs.lselect -text "         Selection:" ] -row 0 -column $::blobulator::text2Column -sticky e
+	grid [entry $::blobulator::blobs.select -width $::blobulator::atomselectWidth -textvariable ::blobulator::select] -row 0 -column $::blobulator::textVariableColumn -columnspan 1  -sticky w
+	grid [checkbutton $::blobulator::blobs.checkSelect -text "Set to blobulate chain by atomselect" \
+	-variable ::blobulator::checkForSelect ]  -row 0 -column $::blobulator::checkBoxColumn -columnspan 2 
+
+	#grid [columnconfigure $blobulator::blobs $::blobulator::checkBoxColumn -uniform]
+
+	#Hydropathy Scale grids
+	grid [label $::blobulator::blobs.t2 -text "         Hydropathy Scale:                                                              " -height 2] -row 2 -column $::blobulator::text2Column -columnspan 2
+	grid [checkbutton $::blobulator::blobs.check -text "Auto Updates Hydrophobicity         " \
+	 -variable ::blobulator::checkForUpdate -command {::blobulator::blobulationSlider }] -row 2 -column $::blobulator::checkBoxColumn -columnspan 2 -sticky w
+	grid [ttk::combobox $::blobulator::blobs.dmnu2  -textvariable ::blobulator::hydropathyScaleDictionaryList -width $::blobulator::dropDownMenuWidth \
+	 -values [list $::blobulator::hydropathyScale1 $::blobulator::hydropathyScale2 $::blobulator::hydropathyScale3] -state readonly] -pady 6 -row 2 -column $::blobulator::dropDownColumn -columnspan 2 -sticky w
```

Comment:
scale1? scale2? I don't understand this line

Reply:
These scales are string values with the hydrophobicity scale names. This generates a list that the drop down menu will use.

Reply:
Made a new issue for organization: #551